opentracing-api/src/main/java/io/opentracing/Span.java

```diff
@@ -13,16 +13,132 @@
  */
 package io.opentracing;
 
+import java.util.Map;
+
 /**
- * Represents an in-flight Span that's <strong>manually propagated</strong> within the given process. Most of
- * the API lives in {@link BaseSpan}.
- *
- * <p>{@link Span}s are created by the {@link Tracer.SpanBuilder#startManual} method; see {@link ActiveSpan} for
- * a {@link BaseSpan} extension designed for automatic in-process propagation.
+ * {@link Span} represents the OpenTracing specification's Span contract.
  *
- * @see ActiveSpan for automatic propagation (recommended for most intstrumentation!)
+ * @see Scope
+ * @see ScopeManager
+ * @see Tracer.SpanBuilder#startManual()
+ * @see Tracer.SpanBuilder#startActive()
  */
-public interface Span extends BaseSpan<Span> {
+public interface Span {
```

Comment:
Not really relevant, but I'm happy to see BaseSpan and other forms of inheritance leaving the API, it makes it cleaner across languages.